packages/components/src/ui/data-table-filter/core/filters.ts

```
@@ -0,0 +1,397 @@
+
+  if (column.options) {
+    return column.options;
+  }
+
+  const filtered = data.flatMap(column.accessor).filter((v): v is NonNullable<TVal> => v !== undefined && v !== null);
+
+  let models = uniq(filtered);
+
+  if (column.orderFn) {
+    models = models.sort((m1, m2) =>
+      column.orderFn!(m1 as ElementType<NonNullable<TVal>>, m2 as ElementType<NonNullable<TVal>>),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)